crates/oci-distribution/src/lib.rs

```diff
@@ -32,11 +33,15 @@ type OciResult<T> = Result<T, failure::Error>;
 /// unless you are sure that the remote registry does not require Oauth2.
 pub struct Client {
     token: Option<RegistryToken>,
+    client: reqwest::Client,
 }
 
 impl Default for Client {
     fn default() -> Self {
```

Comment:
In my opinion, this makes more sense as a `new` function rather than a `Default` implementation.

Reply:
Done.